### PR TITLE
Adds lasagna

### PR DESCRIPTION
**Pull request by Ktlwjec**
https://forums.yogstation.net/index.php?threads/add-lasagna.16382/
Took the sprite from parth here.

:cl:  Ktlwjec, Partheo
rscadd: Adds a new crafting recipe: Lasagna! 
imageadd: Lasagna!
/:cl:

### CODE REVIEW

File: yogstation/code/modules/food_and_drinks/recipes/tablecraft/recipies_spaghetti.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/datum/crafting_recipe/food/lasagna
+	name = "Lasagna"
+	reqs = list(
+		/obj/item/reagent_containers/food/snacks/spaghetti/boiledspaghetti = 2,
```
Comment (alexkar598) [inline yogstation/code/modules/food_and_drinks/recipes/tablecraft/recipies_spaghetti.dm:4]: Since when does lasagna take spaghetti to make

Comment (Ktlwjec) [reply yogstation/code/modules/food_and_drinks/recipes/tablecraft/recipies_spaghetti.dm:4]: It's this or I made it need tomato pasta (SPAGHETTI and tomato to make that). I chose to just use the ingredients instead.

**Comment by Kmc2000** [comment]
You are perfection.

**Review by alexkar598 — APPROVED** [review]

File: yogstation/code/modules/food_and_drinks/recipes/tablecraft/recipies_spaghetti.dm (hunk 1 above)
Comment (TacticalSpacePilot) [reply yogstation/code/modules/food_and_drinks/recipes/tablecraft/recipies_spaghetti.dm:4]: I feel like 1 spaghetti would be better resource-wise.

**Pull request merged**